wrappers/here.rb

```diff
@@ -92,7 +92,7 @@ def route(locs, dimension, departure, arrival, language, with_geometry, options
         routeAttributes: 'summary' + (with_geometry ? ',shape' : '') + (here_strict_restriction(options[:strict_restriction]) == 'soft' ? ',notes' : ''),
         truckType: @mode == 'truck' ? 'truck' : nil,
         trailersCount: options[:trailers], # Truck routing only, number of trailers.
-        limitedWeight: options[:weight], # Truck routing only, vehicle weight including trailers and shipped goods, in tons.
+        limitedWeight: options[:weight].to_i, # Truck routing only, vehicle weight including trailers and shipped goods, in tons.
```

Comment:
I don't think this change is needed, how to call here with a float in grape API?